<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cancer Recognition System</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>4</number>
       </property>
       <widget class="QTreeView" name="treeView">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="originalLungLabel">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>8388607</height>
                </size>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="toolTipDuration">
                <number>-1</number>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="segmentedLungLabel">
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="binaryLungLabel">
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="featuresLungLabel">
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1166</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAnalyze_Original_Image"/>
    <addaction name="actionBinary_Image"/>
    <addaction name="actionAnalyze_Segmented_Image"/>
    <addaction name="actionAnalyze_Features"/>
    <addaction name="separator"/>
    <addaction name="actionImage_Classification"/>
    <addaction name="actionExport_Keypoints"/>
    <addaction name="separator"/>
    <addaction name="actionTrain_SVM"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionImport_Image">
   <property name="text">
    <string>Import Image</string>
   </property>
  </action>
  <action name="actionExport_Keypoints">
   <property name="text">
    <string>Export Keypoints</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImage_Classification">
   <property name="text">
    <string>Image Classification</string>
   </property>
  </action>
  <action name="actionBinary_Image">
   <property name="text">
    <string>Analyze Binary Image</string>
   </property>
  </action>
  <action name="actionAnalyze_Original_Image">
   <property name="text">
    <string>Analyze Original Image</string>
   </property>
  </action>
  <action name="actionAnalyze_Segmented_Image">
   <property name="text">
    <string>Analyze Segmented Image</string>
   </property>
  </action>
  <action name="actionAnalyze_Features">
   <property name="text">
    <string>Analyze Features </string>
   </property>
  </action>
  <action name="actionTrain_SVM">
   <property name="text">
    <string>Train SVM</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
